app/models/public_body.rb

```diff
@@ -702,9 +709,9 @@ def self.popular_bodies(locale)
                         joins(:translations)
       else
         bodies = where("public_body_translations.locale = ?
-                        AND public_bodies.url_name in (?)",
-                        underscore_locale, body_short_names).
-                  joins(:translations)
+                        AND public_body_translations.url_name in (?)",
+                        underscore_locale, body_short_names
+                      ).joins(:translations)
```

Comment:
specs? or commit message saying what specs this was breaking and how the change fixes it